static/src/javascripts/projects/common/modules/experiments/tests/visitor-frequency-quick-survey.js

```diff
@@ -0,0 +1,108 @@
+define([
+    'bean',
+    'fastdom',
+    'lodash/functions/debounce',
+    'common/utils/config',
+    'common/utils/cookies',
+    'common/utils/storage',
+    'common/utils/private-browsing',
+    'text!common/views/experiments/quick-survey.html'
+], function (
+    bean,
+    fastdom,
+    debounce,
+    config,
+    cookies,
+    storage,
+    privateBrowsing,
+    quickSurvey
+) {
+    return function () {
+        this.id = 'VisitorFrequencyQuickSurvey';
+        this.start = '2016-06-13';
+        this.expiry = '2016-06-21';
+        this.author = 'Kate Whalen';
+        this.description = 'Add a single question survey to the submeta section of article pages';
+        this.audience = 0;
+        this.audienceOffset = 0;
+        this.successMeasure = 'Obtain a data-set on how often users visit the Guardian';
+        this.audienceCriteria = 'All users';
+        this.dataLinkNames = 'impressions frequency survey';
+        this.idealOutcome = '';
+
+        this.canRun = function () {
+            return config.page.contentType == 'Article' && checkCookies('GU_FI');
+        };
+
+        function renderQuickSurvey() {
+            // this cannot be wrapped in a fastdom function; the disable function breaks
+            var article = document.getElementsByClassName('content__article-body')[0];
+            var firstChild = article.firstChild;
+            var survey = document.createElement('div');
+            survey.innerHTML = quickSurvey;
+            article.insertBefore(survey, firstChild);
+        }
+
+        function disableRadioButtons(buttonClassName) {
+            var radioButtons = document.getElementsByClassName(buttonClassName);
+            for (var i = 0; i < radioButtons.length; i++) {
+                    radioButtons[i].disabled=true;
+                }
+        }
+
+        function surveyFadeOut() {
+            var surveyTextbox = document.getElementById('surveyTextbox');
+            fastdom.write(function () {
+                surveyTextbox.style.cssText += 'visibility:hidden;opacity:0;transition:visibility 0s ease-in 0.5s,opacity 0.5s linear';
+            });
+        }
+
+        function thankyouFadeIn() {
+            var surveyThanks = document.getElementsByClassName('impressions-survey__thanks');
```

Comment:
Try to use `js-` prefixed classes for getting elements as it separates style from behavioural classes.